src/components/system-security-plans/__tests__/ExportOfferingItemsDialog.spec.ts

```
@@ -0,0 +1,260 @@
+      global: { stubs },
+    });
+
+    await findButton(wrapper, 'Remove').trigger('click');
+    await flushPromises();
+
+    expect(deleteMock).toHaveBeenCalledWith(
+      '/api/oscal/system-security-plans/ssp-1/export-offerings/offering-1/items/item-1',
+    );
+    const emitted = wrapper.emitted('updated');
+    expect(emitted).toBeTruthy();
+    expect(emitted![emitted!.length - 1][0]).toEqual([]);
```

ESLint

[error] Use .at(-1) instead of [emitted!.length - 1]

(e18e/prefer-array-at)